frappe/desk/form/load.py

```diff
@@ -50,7 +50,11 @@ def getdoc(doctype, name, user=None):
 
 	doc.add_seen()
 	set_link_titles(doc)
-	frappe.response.docs.append(doc)
+	if frappe.response.docs:
+		frappe.response.docs.append(doc)
+	else:
+		frappe.response.docs = [doc]
+
```

Comment:
Did you get any clue as to why [it fails only in Postgres build](https://github.com/frappe/frappe/runs/5354635299?check_suite_focus=true#step:14:1095)?

Reply:
Not sure. Looks like an empty list is set in the beginning. Looking into it. 
https://github.com/frappe/frappe/blob/develop/frappe/__init__.py#L193

Reply:
It is failing on [mariadb as well](https://github.com/frappe/frappe/runs/5355426141?check_suite_focus=true). @surajshetty3416